File: LicentaUI/Models/ReviewFilmModel.cs
﻿using System;

namespace LicentaUI.Models
{
    public class ReviewFilmModel
    {
        public string IdReview { get; set; }

        public string IdUser { get; set; }

        public string IdFilm { get; set; }

        public int? Grade { get; set; }

        public string Review { get; set; }

        public Status Status { get; set; }

        public string Title { get; set; }

        public string Author { get; set; }

        public string Description { get; set; }

        public DateTime RelaseDate { get; set; }

        public string PrequelID { get; set; }

        public string SequelID { get; set; }

        public Genre Genre { get; set; }
    }
}

File: LicentaAngular/src/app/models/review-film-model.ts
import { Status } from './status';
import { Genre } from './genre';
import { Rating } from './rating';

export interface ReviewFilmModel {
    idReview: string;
    idUser: string;
    username: string;
    idFilm: string;
    grade: number;
    review: string;
    status: Status;
    film: {
        title: string;
        director: string;
        description: string;
        relaseDate: Date;
        prequeslId: string;
        sequelId: string;
        genre: Genre;
        rating: Rating;
        lenght: number;
        picture: string;
    }
}
